frontend/src/pages/components/ConversationPanel.jsx

```diff
@@ -4,43 +4,98 @@ import styles from '../../styles.module.css';
 import createMuiTheme from "@mui/material/styles";
 import { useState } from 'react'
 
-function ConversationPanel({setMessages}) {
-    const [users, setUsers] = React.useState(["Adam", "Bob", "Claire"]);
-    const mockData = {"Adam": [{'username': 'Adam', 'message': 'I\'m Adam'}],
-                      "     ": [{'username': 'Bob', 'message': 'I\'m Bob'}],
-                      "Claire": [{'username': 'Claire', 'message': 'I\'m Claire'}]};
+function ConversationPanel({setConversationMessages}) {
+    const [chats, setChats] = React.useState([]);
+    // const mockData = {"Adam": [{'username': 'Adam', 'message': 'I\'m Adam'}],
+    //                   "     ": [{'username': 'Bob', 'message': 'I\'m Bob'}],
+    //                   "Claire": [{'username': 'Claire', 'message': 'I\'m Claire'}]};
+    const [currentChat, setCurrentChat] = React.useState("");
+    const [messages, setMessages] = React.useState([]);
     // Endpoint for <id> chat
     // Endpoint for all chat name
     
     
     useEffect(() => {
-        const headers = new Headers()
-        headers.set('Content-Type', 'application/json');
-        headers.set('x-access-token', sessionStorage.getItem('token'));
-        fetch('http://localhost:5000/user', {
-            headers: headers,
-            method: 'GET'
-        }).then((response) => {
-            if (response.ok) 
-                return response.json();
-            else 
-                throw response
-            }
-        ).then((response) => {
-            if (response.users) {
-                setUsers(response.users);
-                return response.users;
+        //let chatroomList = []
+        let headers = new Headers();
+        //headers.set('Content-Type', 'application/json');
+        //headers.set('Accept', 'application/json');
+        headers.set('x-access-token', sessionStorage.token);
+        
+        fetch('http://localhost:5000/chat', {
```

Comment:
I suppose same with this, I believe that the data should already be in the local state at this point. We can take a closer look later.